### PR TITLE
Move theming support from Beta to Main

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #53 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move theming support for the auro-avatar component from beta to main, enabling customization through design tokens. Refactor color and size tokens for better maintainability and flexibility. Update documentation to reflect these changes and adjust tests to ensure correct functionality.

New Features:
- Introduce theming support for the auro-avatar component, allowing customization through design tokens.

Enhancements:
- Refactor color tokens to use tier 3 tokens for improved consistency and maintainability.
- Replace hard-coded size values with design tokens to enhance flexibility and scalability.

Documentation:
- Add documentation for the new theming support, including code samples for restyling the auro-avatar component.

Tests:
- Update test to verify the correct image source for the auro-avatar component.

### CODE REVIEW

File: src/tokens.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/tokens.scss:1]: Not adding the comment at this time.

File: demo/api.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ The auro-avatar component is hidden from screen readers by default. The placemen
 <!-- AURO-GENERATED-CONTENT:END -->
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply demo/api.md:154]: This is correct.

**Comment by DukeFerdinand** [comment]
Hmmm this seems to be broken (at least running locally). I would imagine there's supposed to be some examples rendering here?

It's building the code correctly as far as I can tell, but it's not actually rendering any image to the page.

Screenshot 2024-09-18 at 11 08 23 AM

**Comment by jordanjones243** [comment]
> Hmmm this seems to be broken (at least running locally). I would imagine there's supposed to be some examples rendering here?
> 
> It's building the code correctly as far as I can tell, but it's not actually rendering any image to the page.
> 
> Screenshot 2024-09-18 at 11 08 23 AM

Just checked my localhost and it works for me!


Screen Shot 2024-09-18 at 2 53 53 PM

**Review by jason-capsule42 — APPROVED** [review]

**Review by DukeFerdinand — APPROVED** [review]
Update - I got things to run locally so this looks good

**Pull request merged**